ewc-rfc001-issue-verifiable-credential.md

```diff
@@ -625,6 +745,12 @@ This request is made with the following query params:
    <td>Wallet-generated secure random token used to validate the original <code>code_challenge</code> provided in the initial Authorization Request
    </td>
   </tr>
+   <tr>
+   <td><code>redirect_uri</code>
+   </td>
+   <td>For redirection of the response as per <a href=#https://www.rfc-editor.org/rfc/rfc6749.html#section-3.1.2">IETF RFC6749 Section 3.1.2</a>
+   </td>
+  </tr>
 </table>
 
 ### 3.7.2 Pre-authorised code flow
```

Comment:
I believe the token request as described here, without a client id, requires `pre-authorized_grant_anonymous_access_supported` to be true in the oauth server metadata. But this isn't discussed in the oauth server issuer metadata section (3.4).

similarly should probably discuss `tokenEndpointAuthorizationMethodsSupported`. it needs to contain `none` if no authorization is performed at the token endpoint.

Reply:
Fixed the first by specifying `authorized_grant_anonymous_access_supported` in authorisation server metadata. 

@lalc For the second one, I am not sure, if the value can be `none` as per the specification (RFC 8414):

![image](https://github.com/user-attachments/assets/f04f6c3a-263e-4cdd-ba4c-3fd7c1c5a321)

Reply:
`none` is in the linked spec RFC7591 https://www.rfc-editor.org/rfc/rfc7591#section-2

Reply:
> `none` is defined in the linked spec RFC7591 https://www.rfc-editor.org/rfc/rfc7591#section-2

But that refers to the `token_endpoint_auth_method` field and you referred to the `token_endpoint_auth_methods_supported` earlier?

Reply:
Also, it is from the perspective of the OAuth client informing OAuth AS right ?

Reply:
it's all in the screenshot you posted. `token_endpoint_auth_methods_supported` contains is a list of client authentication methods support by this token endpoint. The token endpoint auth methods are defined in Section 2 of RFC7591 which is linked in that screenshot from 8414. 8414 just links to 7591 as the source for the auth methods.